src/main/bash/sdkman-help.sh

```diff
@@ -35,7 +35,7 @@ function __sdk_help {
 	__sdkman_echo_no_colour "       offline           [enable|disable]"
 	__sdkman_echo_no_colour "       selfupdate        [force]"
 	__sdkman_echo_no_colour "       update"
-	__sdkman_echo_no_colour "       flush             <broadcast|archives|temp>"
+	__sdkman_echo_no_colour "       flush             <broadcast|archives|temp|all>"
```

Comment:
If `all` is redundant, we could make this:

    flush           [broadcast|archives|temp]

BTW, I'll remove the flushing of broadcast in a subsequent PR as it is actually no longer required.

Reply:
Oh sorry I should have read further :see_no_evil: